deployer/src/main/java/com/percussion/rx/config/impl/PSConfigStatusMgr.java

```diff
@@ -138,13 +139,16 @@ public List<PSConfigStatus> findConfigStatus(String nameFilter)
       List<PSConfigStatus> cfgStatusList = null;
       Session session = getSession();
 
-         Criteria criteria = session.createCriteria(PSConfigStatus.class);
-         criteria.add(Restrictions.like("configName", nameFilter).ignoreCase());
-         criteria.addOrder(Order.asc("configName"));
-         criteria.addOrder(Order.desc("dateApplied"));
-         cfgStatusList = criteria.list();
 
-      return cfgStatusList;
+
+      CriteriaBuilder builder = getSession().getCriteriaBuilder();
+      CriteriaQuery<PSConfigStatus> criteria = builder.createQuery(PSConfigStatus.class);
+      Root<PSConfigStatus> critRoot = criteria.from(PSConfigStatus.class);
+      criteria.where(builder.like(critRoot.get("configName"),nameFilter));
+      criteria.orderBy(builder.asc(critRoot.get("configName")));
```

Comment:
This is missing the case insensitive option - should be lowercasing or uppercasing to match.